### PR TITLE
Lint Configuration

### PR DESCRIPTION
**Pull request by Nyan-Thar-Myint**

### CODE REVIEW

File: .github/workflows/buildsrcipt.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    branches: [ "main", "dev" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: set up JDK 11
```
Comment (Nyan-Thar-Myint) [inline .github/workflows/buildsrcipt.yml:14]: Naming

File: .github/workflows/buildsrcipt.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
+name: Android CI
+
+on:
+  pull_request:
+    branches: [ "main", "dev" ]
```
Comment (Nyan-Thar-Myint) [inline .github/workflows/buildsrcipt.yml:5]: Format

File: .github/workflows/buildsrcipt.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+      - name: set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'temurin'
+          cache: gradle
+
+      - name: Grant execute permission for gradlew
+        run: chmod +x gradlew
+      - name: Build with Gradle
```
Comment (Nyan-Thar-Myint) [inline .github/workflows/buildsrcipt.yml:23]: Line break

File: .github/workflows/buildsrcipt.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+name: Android CI
```
Comment (Nyan-Thar-Myint) [inline .github/workflows/buildsrcipt.yml:1]: Rename

**Review by Nyan-Thar-Myint — COMMENTED** [review]
Format and rename buildscript.yml

**Review by Nay-Myo-Min — APPROVED** [review]

**Pull request merged**